src/pages/hotel/booking/book-header/BookHeader.scss

```diff
@@ -0,0 +1,62 @@
+.book-header-container {
+  padding: 1px;
+  margin: 0rem 10rem 0rem 8.5rem;
+}
+.book-items {
+  position: relative;
+  flex: 1;
+  background: white;
+  margin: 20px 0px -10px 0px;
+}
+.indicator {

```

Comment:
Same name used in the swiper. So default swiper styles changed now. Can you change the name ? or else need to use parent class path as well with this.

![image](https://user-images.githubusercontent.com/110229619/205553682-a32a13fc-34ee-4cba-9a3c-d60d10b80786.png)

Reply:
Changed the indicator name according to page.